addons/auth_passkey_portal/security/security.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.rule" id="rule_auth_passkey_key_portal">
+        <field name="name">Passkeys: Portal users can only access own Passkeys</field>
+        <field name="model_id" ref="auth_passkey.model_auth_passkey_key"/>
+        <field name="groups" eval="[(4, ref('base.group_portal'))]"/>
+        <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        <field name="perm_unlink" eval="1"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_create" eval="1"/>
+    </record>
+
+    <record model="ir.rule" id="rule_auth_passkey_key_create_portal">
+        <field name="name">Passkeys: Portal users can only modify their own passkey creation requests</field>
+        <field name="model_id" ref="auth_passkey.model_auth_passkey_key_create"/>
+        <field name="groups" eval="[(4, ref('base.group_portal'))]"/>
+        <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        <field name="perm_unlink" eval="1"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_create" eval="1"/>
+    </record>
```

Comment:
I see there is no rule for `model_auth_passkey_key_create` in the `auth_passkey` module at all.
There is nothing critical in that model so it doesn't matter.

But if you add a security rule, then I would add it in `auth_passkey`, and would add `group_portal` there directly.